frontend/src/app/patients/ManagePatients.test.tsx

```diff
@@ -0,0 +1,163 @@
+import { MockedProvider, MockedProviderProps } from "@apollo/client/testing";
+import {
+  fireEvent,
+  render,
+  screen,
+  waitForElementToBeRemoved,
+} from "@testing-library/react";
+import { MemoryRouter } from "react-router";
+
+import ManagePatients, {
+  patientQuery,
+  patientsCountQuery,
+} from "./ManagePatients";
+
+const TestContainer: React.FC = ({ children }) => (
+  <MemoryRouter>
+    <MockedProvider mocks={mocks}>
+      <>{children}</>
+    </MockedProvider>
+  </MemoryRouter>
+);
+
+describe("ManagePatients", () => {
+  it("renders a list of patients", async () => {
+    render(
+      <TestContainer>
+        <ManagePatients
+          activeFacilityId="a1"
+          canEditUser={true}
+          canDeleteUser={true}
+          isAdmin={false}
+        />
+      </TestContainer>
+    );
+    expect(await screen.findByText(patients[0].lastName, { exact: false }));
+    expect(await screen.findByText(patients[1].lastName, { exact: false }));
+    expect(await screen.findByText(patients[2].lastName, { exact: false }));
+  });
+  it("filters a list of patients", async () => {
+    jest.useFakeTimers();
+    render(
+      <TestContainer>
+        <ManagePatients
+          activeFacilityId="a1"
+          canEditUser={true}
+          canDeleteUser={true}
+          isAdmin={false}
+        />
+      </TestContainer>
+    );
+    expect(await screen.findByText(patients[0].lastName, { exact: false }));
+    const btn = await screen.findByText("Filter", { exact: false });
+    fireEvent.click(btn);
+    const input = await screen.findByLabelText("Person");
+    fireEvent.change(input, { target: { value: "Al" } });
+    await waitForElementToBeRemoved(() =>
+      screen.queryByText("Abramcik", { exact: false })
+    );
+    expect(await screen.findByText(patients[1].lastName, { exact: false }));
+    expect(await screen.findByText(patients[2].lastName, { exact: false }));
```

Comment:
Thanks for adding this! Glad we are starting to flesh out unit tests on the frontend!